Extensions/postarchive.js

```diff
@@ -1130,7 +1156,7 @@ XKit.extensions.postarchive = {
 		var blog_url = m_post.owner;
 
 		if (!blog_url) {
-			blog_url = window.location.href.split('%2F')[2].split('.')[0];
+			blog_url = window.location.href.split('/')[2].split('.')[0];
```

Comment:
Hmm. And there's no other way to get the window to show up—other then width: 100%?